presto-hive/src/main/java/com/facebook/presto/hive/util/StatisticsUtils.java

```diff
@@ -16,15 +16,19 @@
 import com.facebook.presto.hive.HiveBasicStatistics;
 import com.facebook.presto.hive.metastore.Partition;
 import com.facebook.presto.hive.metastore.Table;
+import com.google.common.primitives.Ints;
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.OptionalInt;
 
 import static com.google.common.base.Preconditions.checkArgument;
 import static java.util.Collections.unmodifiableMap;
 
 public class StatisticsUtils
 {
+    public static final String NUM_PARTITIONS = "numPartitions";
```

Comment:
nit: I would keep `private StatisticsUtils() {}` first, since it's not a real constructor, just a utility class marker